moveit_core/robot_state/include/moveit/robot_state/robot_state.h

```diff
@@ -1647,7 +1659,9 @@ class RobotState
    * the body positions will get corrupted.  You need to make a fresh
    * copy of the AttachedBody object for each RobotState you attach it
    * to.*/
-  void attachBody(AttachedBody* attached_body);
+  [[deprecated("Deprecated use the overload requiring an unique_ptr instead")]] void
+  attachBody(AttachedBody* attached_body);
+  // TODO Remove this function
```

Comment:
```suggestion
```
either remove this or add some reasoning when to remove this (ie some version of MoveIt 2)

Reply:
Removed, as @rhaschke mentioned that removal of deprecated functions is automatic.